### ISSUE TITLE
Issues with checkboxGroup using custom render function

### ISSUE THREAD

**Issue by davechisholm** [grommet/grommet#6828]
When using CheckboxGroup with a custom render function, the output follows this structure 
```
<label label="<yourcheckboxLabel>">
   <div>
      <input type="checkbox" ... />
      <output of your render function here>
   </div>
   <span><yourcheckboxLabel></span>
</label>
```

A few issues we're having with this
We need to manually hide the auto-generated span
The 'label' atttribute on the 'label' elem, i don't think is valid html? Should be aria-label
Our tests are not able to use getByLabelText(), probably because of the above, and because we have to hide the span

Any thoughts/workarounds welcome - eg is there any way to modify the wrapper label element, or remove the span

**Comment by jcfilben** [comment]
It looks like the label in the label element is just grabbing whatever is passed to the `label` prop in CheckBox, so if we pass an element our label ends up being [object Object]. We may have to specify a way to pass a pure text label that is used here instead when you want to display a custom element label.

**Comment by davechisholm** [comment]
I think the issue is that 'label' is not the correct property - it should be 'arial-label'

**Comment by jcfilben** [comment]
Closing, no longer seeing this issue. Feel free to re-open if needed
